sbin/patchman

```diff
@@ -579,6 +594,9 @@ def process_args(args):
     if args.errata:
         update_errata(args.force)
         showhelp = False
+    if args.killhost:
```

Comment:
This if statement also needs to be changed to `delete_host`?

Reply:
Should be fixed now.